docs/topics/logging.txt

```diff
@@ -261,12 +265,37 @@ First, here's a simple configuration which writes all request logging from the
 If you use this example, be sure to change the ``'filename'`` path to a
 location that's writable by the user that's running the Django application.
 
-Second, here's an example of a fairly complex logging setup, configured using
+Second, here's an example of how to make the logging system print all of
+Django's logging to the console based on the value of an environment variable.
+It overrides the fact that ``django.request`` and ``django.security`` don't
+propagate their log entries. It may be useful during local development. Set the
+environment variable ``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging
```

Comment:
I'm not sure what the point of having the `DJANGO_LOG_LEVEL` environment variable in this example is; seems like needless added complexity that's beside the point of the example. I'd just remove the env var and set the level to `INFO`.

Also, the description of the effect of the env var is wrong here -- with the code as below, if you don't set the env var, you'll see anything of level `INFO` or higher. You can just set the env var to change that threshold level.